hm_pyhelper/util/sentry.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict, Union
+from collections import Counter
+
+EVENT_COUNT_KEY = "event_count"
+SUPPRESSION_COUNT_KEY = "suppression_count"
+
+# per session event counters, didn't feel the need to persist them
+event_counters: Dict[str, Counter] = {}
+
+# maximum suppression interval. It grows exponentially till this value is reached.
+max_suppression_count = 128
+
+
+def sentry_fingerprint(hints: Dict) -> Union[str, None]:
+    """ return sha256 of log_record as fingerprint"""
+
+    if 'log_record' not in hints:
+        return None
+
+    log_record = hints.get('log_record', None)
+    fingerprint = f"{log_record.filename}:{log_record.lineno}:{log_record.funcName}"
```

Comment:
nit: funcname for consistency

Reply:
not our data structure, it comes from Logging module.